core/src/test/scala/org/apache/spark/storage/BlockManagerMasterSuite.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.storage
+
+import org.junit.Assert.assertTrue
+
+import org.apache.spark.{SparkConf, SparkFunSuite}
+
+class BlockManagerMasterSuite extends SparkFunSuite {
```

Comment:
Add tests to `BlockManagerSuite` should be enough?

Reply:
Yes. I considered that, but BlockManagerSuite looks misleading to me because this is `BlockManagerManger`. If there is a more broader suite, it could be a candidate.

Reply:
I actually think it should be ok because we always test `BlockManagerMaster` together with `BlockManager`. And there're already some tests which test `BlockManagerMaster` only, e.g.

- `query locations of blockIds`

- `SPARK-30594: Do not post SparkListenerBlockUpdated when updateBlockInfo returns false`

Reply:
Sorry, but I don't think so. In your example, `query locations of blockIds` is just do mocking instead of testing.
```
test("query locations of blockIds") {
    val mockBlockManagerMaster = mock(classOf[BlockManagerMaster])
```

Reply:
Ok, fine.

Reply:
Also, `BlockManagerSuite` is already large with 1805 lines. In general, we had better have a proper UT case.